tools/bpf2c/bpf2c.cpp

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) eBPF for Windows contributors
 // SPDX-License-Identifier: MIT
 
+#include "../../libs/shared/hash.h"
```

Comment:
Rather than putting the path here, it's better to put it in the include path in the vcxproj file.

Reply:
updated